go-eicio/eicio-ls/main.go

```diff
@@ -8,7 +8,7 @@ import (
 	"log"
 	"os"
 
-	"github.com/decibelCooper/eicio/go-eicio"
+	"github.com/decibelcooper/eicio/go-eicio"
```

Comment:
this may be ok on macOS where the filesystem is case-insensitive (well, it's complicated), but on linux, that may fail.

also, you may end up (in a 3rd-party program) with 2 copies of the code: one from under `github.com/decibelcooper/eicio` and another one from under `github.com/decibelCooper/eicio`...

to make sure everybody imports the package from `github.com/decibelcooper/eicio`, you should probably add:

```go
package eicio // import "github.com/decibelcooper/eicio/go-eicio"
```
in one of the files under `.../eicio/go-eicio` (_e.g.:_ `event.go`)

Reply:
Hey @sbinet,

Thanks for the input.  I agree that this is a concern.  I like your solution: does this comment show up in godoc?  Right now godoc.org is redirecting .../decibelcooper/... to .../decibelCoooper/...

I don't know how to solve this.  I just felt that it was a mistake in the first place to have a non-lowercase character in the first place, and I thought that I should go ahead and make the change.

Reply:
Also, I have now changed the casing on my account name to all lower.  After some refreshing in godoc.org, this has now fixed the documentation there to be all lower-case, and redirect to the lower-case if looking up the upper-case version.

Reply:
the directive:

```go
package eicio // import "github.com/decibelcooper/eicio/go-eicio"
```

is actually enforced at the toolchain level, so yeah, godoc would show it.

Reply:
Excellent!